### ISSUE TITLE
Using sql.js in typescript

### ISSUE THREAD

**Issue by twotree** [kripken/sql.js#151]
I've installed sql.js and tried using
  import {SQL} from 'sql.js/js/sql'; 
to no avail.  Do I need to use require.js?  Is there an example somewhere showing how to use it with TypeScript?

**Comment by variab1e** [comment]
@twotree 
I am using it with TypeScript, here is a [gist](https://gist.github.com/variab1e/fe0ca29c946cd46c7bccdab56ceaff0e)

**Comment by Taytay** [comment]
@erichiller  : Can you re-link to your gist? I'm getting a 404.

**Comment by kalmargabor** [comment]
@twotree @Taytay I've found it in Google: https://gist.github.com/erichiller/fe0ca29c946cd46c7bccdab56ceaff0e

**Comment by pando85** [comment]
Any updates?

**Comment by FrancescoBorzi** [comment]
@brodybits @lovasoa  are there any plans to support TypeScript?

**Comment by lovasoa** [comment]
Yes, I'd like type annotations to be added, after #349 is merged. Help is welcome

**Comment by jjhbw** [comment]
`@types/sql.js` worked pretty well for my use case

**Comment by albertogasparin** [comment]
If anybody else has issues with `import ... from '@jlongster/sql.js'` not getting typed, you can use npm alias to let TS properly match the `sql.js` types.
```
dependencies: {
  ...
  "@types/jlongster__sql.js": "npm:@types/sql.js@latest",
  ...
}
```
